deploy/docs/Best_Practices.md

```diff
@@ -25,6 +25,11 @@
 - [Get logs not available on stdout](#get-logs-not-available-on-stdout)
 - [Adding custom fields](#adding-custom-fields)
 - [Using custom Kubernetes API server address](#using-custom-kubernetes-api-server-address)
+- [OpenTelemetry queueing and batching](#opentelemetry-queueing-and-batching)
```

Comment:
While this is nice I wonder if we shouldn't change this to have 3 levels with OT general  heading, then queueing and batching then all the topics mentioned below.

Probably as a separate PR but still :)

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/2136